frontend/src/core/components/AppOrderSummary/AppOrderSummary.js

```diff
@@ -27,7 +27,7 @@ class AppOrderSummary extends Component {
         ))}
         <div className="item">
           <p>Current balance</p>
-          <Form.Control value={balance} onChange={balanceOnChange} />
+          <Form.Control value={balance} onChange={balanceOnChange} readOnly />
```

Comment:
@justaname94 The user needs edit the value, to adjust the amount to checkout, so, this field cannot be read only.